src/Microsoft.Dnx.Tooling/Building/BuildOptions.cs

```diff
@@ -23,8 +24,15 @@ public class BuildOptions
         public BuildOptions()
         {
             Configurations = new List<string>();
-            TargetFrameworks = new List<string>();
             ProjectPatterns = new List<string>();
         }
+
+        public void AddFrameworkMonikers(IEnumerable<string> monikers)
+        {
+            foreach (var moniker in monikers)
+            {
+                TargetFrameworks.Add(FrameworkNameHelper.ParseFrameworkName(moniker), moniker);
```

Comment:
This will throw for duplicate keys - is this the intended behavior (as oppose to ignore duplicates)?